src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, memo, useMemo } from 'react';
+import styled from 'styled-components';
+
+import { gray100 } from '../../core/Colors';
+
+export interface SkeletonProps {
+  style?: React.CSSProperties;
+  className?: string;
+  circle?: boolean;
+  rounded?: boolean;
+  width?: number | string;
+  height?: number | string;
+  count?: number;
+  color?: string;
+  lineHeight?: number;
+}
+
+export const Skeleton: FC<SkeletonProps> = memo(
+  ({ style, className, circle, width, height, count, color, lineHeight, rounded }) => {
+    const content = useMemo(() => {
+      let result = '';
+      for (let i = 0; i < (count || 1); i += 1) {
```

Comment:
```suggestion
      for (let i = 0; i < count; i++) {
```
이렇게 바꾸고 count의 defaultProps를 1로 설정하면 안되나요?

Reply:
그러면 css도 변경해야하는데 해당 코드는 count가 없어도 line-height를 잡도록 유도한 코드라서 저렇게 명시적으로 했으면 좋겠는데 어떠신가요
@john015

Reply:
count가 글자 수 이죠?? 문서에 추가해주세용

Reply:
count가 0 일 때 말하는거면 
```suggestion
    for (let i = 1; i < count; i++) {
```
이렇게 바꾸시고 
```
        <span>F{content}</span>
```
이렇게 하면 되는거 아닌가유?